test/fixtures/xml/usps/domestic_label_response.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<DelivConfirmCertifyV4.0Response>
+  <DeliveryConfirmationNumber><DELIVERY CONFIRMATION NUMBER></DeliveryConfirmationNumber>
```

Comment:
Ah, this is my fault - in the review I said you should use `<LABEL IMAGE REDACTED DUE TO SIZE>` but the angle brackets (`<`/`>`) would mess with the parsing. You can omit those. 😅